enterprise/backend/src/metabase_enterprise/data_editing_public/api.clj

```diff
@@ -20,10 +20,9 @@
    row-or-rows]
   (let [table-id (api/check-404 (t2/select-one-fn :table_id :table_webhook_token :token token))
         rows     (if (map? row-or-rows) [row-or-rows] row-or-rows)]
-    (when (seq rows)
-      (api/check-400 (every? seq rows))
-      (data-editing/insert! table-id rows))
-    {}))
+    (api/check-400 (seq rows) "Please supply at least one row.")
+    (api/check-400 (every? seq rows) "Every row should not be empty.")
+    (data-editing/insert! table-id rows)))
```

Comment:
Intentionally changing the return value?

Should probably test the response in api tests.